tomopy/prep/alignment.py

```diff
@@ -443,3 +455,240 @@ def shift_images(prj, sx, sy):
     prj *= scl
 
     return prj
+
+
+def find_slits_corners_aps_1id(img, 
+                               method='quadrant+', 
+                               medfilt2_kernel_size=3,
+                               medfilt_kernel_size=23,
+                               ):
+    """
+    Automatically locate the slit box location by its four corners.
+
+    Parameters
+    ----------
+    img                   :  np.2darray
+        input 2D images
+    method                :  str,  ['simple', 'quadrant', 'quadrant+'], optional
+        method for auto detecting slit corners
+    medfilt2_kernel_size  :  int, optional 
+        2D median filter kernel size for noise reduction
+    medfilt_kernel_size   :  int, optional 

```

Comment:
I'd simply use kernel_size and explain in-line that it is for the median filter below to shorten the argument name.

p.s. This is also consistent with scipy.signal.medfilt

Reply:
I think the prefix here is necessary as they are intended for different median filters for different purposes.  I am currently working on an adaptive method, which might be able to get away using a universal kernel_size.  However, that is for a different pull request, and I do have a few bugs to figure out before I can make the PR.